### PR TITLE
[Do not merge]: Updated content for getting MIME content of a message.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/message-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/api/message-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/mail-api-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/resources/mail-api-overview.md) | :white_check_mark:Succeeded | |
[concepts/outlook-get-mime-message.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-get-mime-message.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812060834146561-3564%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7fy0Y2xzQGy%2FTi%2FukZWPsGak5kTzaL5fMv%2FQ3ilorj0%3D&st=2018-12-06T08%3A32%3A23Z&se=2019-01-06T08%3A37%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/message-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/api/message-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/mail-api-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/resources/mail-api-overview.md) | :white_check_mark:Succeeded | |
[concepts/outlook-get-mime-message.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-get-mime-message.md) | :white_check_mark:Succeeded | |
[concepts/outlook-mail-concept-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-mail-concept-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812060854205092-3564%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5EMFXwxZZE1iGiSqhdceRL%2FaXXf6Vz%2BtPOLhFW5nw78%3D&st=2018-12-06T08%3A52%3A29Z&se=2019-01-06T08%3A57%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/outlook-get-mime-message.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: "Get MIME content from a message"
+description: "Multipurpose Internet Mail Extensions (MIME) is an industry email standard. You can now use a `$value` segment to get the MIME content of an Outlook message."
+---
+
+# Get MIME content from a message (preview)
+
+Multipurpose Internet Mail Extensions (MIME) is an industry email standard. Many email applications create messages in MIME format and save them in files with the .EML extension.
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md]: FYI, we don't need to spell out MIME: https://worldready.cloudapp.net/Styleguide/Read?id=2812&topicid=38584

File: concepts/outlook-get-mime-message.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+description: "Multipurpose Internet Mail Extensions (MIME) is an industry email standard. You can now use a `$value` segment to get the MIME content of an Outlook message."
+---
+
+# Get MIME content from a message (preview)
+
+Multipurpose Internet Mail Extensions (MIME) is an industry email standard. Many email applications create messages in MIME format and save them in files with the .EML extension. 
+Even though Outlook _does not save_ messages in MIME format, you can now use a `$value` segment to get the MIME content of an Outlook message. You can then save the content in a .EML file 
+and attach the file to records in business systems, such as those for CRM, ERP, and bug tracking. 
+
+> **Important:** This capability is currently available only in the /beta version. Similar to other API in preview status, it is subject to change. Do not use this capability in production apps. For more information, see [versioning and support](versioning-and-support.md).
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md]: Change API to APIs.

File: api-reference/beta/api/message-get.md [hunk 3]
```diff
@@ -49,9 +60,11 @@ GET /users/{id | userPrincipalName}/mailFolders/{id}/messages/{id}?$expand=menti
 ```
 
 ## Optional query parameters
-This method supports the [OData Query Parameters](https://developer.microsoft.com/graph/docs/concepts/query_parameters) to help customize the response.
+This method supports the [OData Query Parameters](/graph/query_parameters) to help customize the response.
```
Comment (Lauragra) [inline api-reference/beta/api/message-get.md]: Don't forget to update the file names to use hyphens, query-parameters.

File: concepts/outlook-get-mime-message.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+## What is MIME?
+
+MIME is a standard used by Internet email to transmit the following types of content via SMTP:
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md]: FYI, no need to cap Internet: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/internet-intranet-extranet

File: concepts/outlook-get-mime-message.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+Even though Outlook _does not save_ messages in MIME format, you can now use a `$value` segment to get the MIME content of an Outlook message. You can then save the content in a .EML file 
+and attach the file to records in business systems, such as those for CRM, ERP, and bug tracking. 
+
+> **Important:** This capability is currently available only in the /beta version. Similar to other API in preview status, it is subject to change. Do not use this capability in production apps. For more information, see [versioning and support](versioning-and-support.md).
+
+## What is MIME?
+
+MIME is a standard used by Internet email to transmit the following types of content via SMTP: 
+
+- plain text message
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md]: Each bullet should start with a cap and no need to end in a period if phrases.

File: concepts/outlook-get-mime-message.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+You can get the MIME representation of a message by appending the `$value` segment when 
+[getting a message](/graph/api/message-get?view=graph-rest-beta): 
+
+<!-- { "blockType": "ignored" } -->
+```http
+GET /me/messages/{id}/$value
+```
+
+The following are typical MIME headers in a message. For more information, see [MIME](https://en.wikipedia.org/wiki/MIME).
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md]: Can we remove the link or find a more authoritative source?

File: concepts/outlook-get-mime-message.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+- `MIME-Version` which indicates the message is MIME-formatted.
```
Comment (Lauragra) [inline concepts/outlook-get-mime-message.md]: We can replace "which" with a dash here for readability: - Indicates that the message....

File: concepts/outlook-get-mime-message.md (hunk 1 above)
Comment (angelgolfer-ms) [reply concepts/outlook-get-mime-message.md]: ok

File: concepts/outlook-get-mime-message.md (hunk 2 above)
Comment (angelgolfer-ms) [reply concepts/outlook-get-mime-message.md]: ok

File: api-reference/beta/api/message-get.md (hunk 3 above)
Comment (angelgolfer-ms) [reply api-reference/beta/api/message-get.md]: And case of the link text.

File: concepts/outlook-get-mime-message.md (hunk 5 above)
Comment (angelgolfer-ms) [reply concepts/outlook-get-mime-message.md]: Right. https://docs.microsoft.com/en-us/style-guide/scannable-content/lists

File: concepts/outlook-get-mime-message.md (hunk 6 above)
Comment (angelgolfer-ms) [reply concepts/outlook-get-mime-message.md]: The wikipedia article is much more readable for those new to MIME. I replaced the reference by an RFC article; I don't think it's as helpful a reference if developers care more about getting the MIME content than understanding MIME in detail.

File: concepts/outlook-get-mime-message.md (hunk 7 above)
Comment (angelgolfer-ms) [reply concepts/outlook-get-mime-message.md]: ok.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[212d316](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3564/commits/212d31636e9411f7efb731f1a197c5a39343b230)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/message-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/api/message-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/mail-api-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/resources/mail-api-overview.md) | :white_check_mark:Succeeded | |
[concepts/outlook-get-mime-message.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-get-mime-message.md) | :white_check_mark:Succeeded | |
[concepts/outlook-mail-concept-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-mail-concept-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C11%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812110151462954-3564%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qEhsjmMDUzzF6g6btJgE9cGaC1CUtjsjlemnyhLvCjk%3D&st=2018-12-11T01%3A50%3A12Z&se=2019-01-11T01%3A55%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[a9bc480](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3564/commits/a9bc480f92ae8b80f5ba752541f4c989bfbcd3a1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/message-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/api/message-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/mail-api-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/resources/mail-api-overview.md) | :white_check_mark:Succeeded | |
[concepts/outlook-get-mime-message.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-get-mime-message.md) | :white_check_mark:Succeeded | |
[concepts/outlook-mail-concept-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-mail-concept-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C11%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812110152062158-3564%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=pK22t4MAFeAzi7g0tY6Losg2Y%2B6x%2FzuHuvkeSx7zlZY%3D&st=2018-12-11T01%3A53%3A35Z&se=2019-01-11T01%3A58%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[bca9a84](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3564/commits/bca9a8490d1b6fafa68a4661dd156d00db76c2fc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/message-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/api/message-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/mail-api-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/api-reference/beta/resources/mail-api-overview.md) | :white_check_mark:Succeeded | |
[concepts/outlook-get-mime-message.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-get-mime-message.md) | :white_check_mark:Succeeded | |
[concepts/outlook-mail-concept-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/ach-mm-content/concepts/outlook-mail-concept-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C27%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903270649498375-3564%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BQ6sgXb4VM%2B5sbbZqmhjn%2BOaCWjAKey4VokoAi9N0fM%3D&st=2019-03-27T06%3A48%3A39Z&se=2019-04-27T06%3A53%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**